src/content_handlers/apt_handler/inc/aduc/apt_parser.hpp

```diff
@@ -29,12 +29,18 @@
  */
 #define ADU_APT_FIELDNAME_VERSION "version"
 
+/**
+ * @brief JSON field name for agentRestartRequired property
+ */
+#define ADU_APT_FIELDNAME_AGENT_RESTART_REQUIRED "agentRestartRequired"
+
```

Comment:
is this documented somewhere?

Reply:
Will work with Val on the doc changes for this.

---
In reply to: [596181802](https://github.com/Azure/iot-hub-device-update/pull/17#discussion_r596181802) [](ancestors = 596181802)